service/internal/server/server.go

```diff
@@ -129,6 +129,89 @@ type CORSConfig struct {
 	AllowCredentials bool     `mapstructure:"allowcredentials" json:"allowcredentials" default:"true"`
 	MaxAge           int      `mapstructure:"maxage" json:"maxage" default:"3600"`
 	Debug            bool     `mapstructure:"debug" json:"debug"`
+
+	// Additive fields - appended to base lists at runtime without replacing defaults
+	AdditionalMethods        []string `mapstructure:"additionalmethods" json:"additionalmethods"`
```

Comment:
I'm good with this, but would `AdditionalAllowed*` be a better convention so the associations are even more closely aligned between `AllowedMethods -> AdditionalAllowedMethods` `AllowedHeaders -> AdditionalAllowedHeaders`?

Reply:
I think `AdditionalExposedHeaders` would stay the same since it's different than `AdditionalAllowedHeaders`, but we can just leave this how it is.